tabular/src/autogluon/tabular/models/image_prediction/image_predictor.py

```diff
@@ -0,0 +1,207 @@
+
```

Comment:
what happens with HPO for this model?

Reply:
Currently image model would fail unless a search space was provided by user (same as TextNN currently)

Reply:
Added default HPO search space, but we probably want to have a better search space in future